pinecone/index.py

```diff
@@ -169,6 +241,37 @@ def _vector_transform(item: Union[Vector, Tuple]):
             **{k: v for k, v in kwargs.items() if k in _OPENAPI_ENDPOINT_PARAMS}
         )
 
+    def upsert_from_dataframe(self,
+                         df,
+                         namespace: str = None,
+                         batch_size: int = 500,
+                         show_progress: bool = True) -> None:
```

Comment:
We should set this to `false` by default we don't want to print out stuff to users stdout without them expecting it to